crates/catalog/rest/testdata/create_table_response.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
How do we feel about testing against the REST Docker container? This will make the testing much easier in the future.

Reply:
Yes, I'll add such integration tests later pr. It takes some effort to setup the testing framework.